### ISSUE TITLE
Dental panoramic detection: Overfitting problem

### ISSUE THREAD

**Issue by gksruf** [pjreddie/darknet#2455]
Hello. I am not an expert in deep learning and seeking for an help. I am trying to customize YOLOv4 with two pathologic lesions on dental panoramic radiographs. Class 1 has total of 455 (384 training set & 71 test set) and Class 2 has total of 1441 images (1223 training set & 218 test set). All dental panoramic images seems similar in general because people have similar bone structure. However, two different lesions are usually located in the apex of the tooth and they are both pretty small in the large panoramic radiograph. Since labeled lesions are small, we cropped the image when training and tested them. However, the resulting graph looks like below:

![image](https://user-images.githubusercontent.com/12556799/123534015-d5418e00-d754-11eb-8525-a7a186f79ffa.png)

It overfits very soon. What should I do to fix this problem? Should I try to change masks, anchors, classes, or num in "yolov4.cfg"? Would it help improve the problem? Especially, I am thinking about changing anchors and I wonder how.
Or should I use YOLOv3 instead of v4? or could I change v4 more shallower using darknet? Please help me solve the problem. Thank you and have a great day!

**Comment by saiteja-kuchipudi** [comment]
Hello @gksruf. Did you solve this problem of overfit? If yes, can you share the way out

**Comment by gksruf** [comment]
> **[saiteja-kuchipudi](/saiteja-kuchipudi) ** c

Half right and half wrong. We could increase the accuracy about 10~15% but cannot fully solve overfitting problem.
I am not sure the way i used work for your case.  Moreover, since the way i used is not simple, it is not easy to explain.
Could you explain your goals and problems?

**Comment by gksruf** [comment]
I tried many option to avoid overfit, but none of them works.
Android용 Outlook<https://aka.ms/ghei36> 받기

________________________________
